onnxscript/function_libs/torch_lib/ops/core.py

```diff
@@ -5342,10 +5342,16 @@ def aten_pdist(self: TensorType, p: float = 2.0) -> TensorType:
     raise NotImplementedError()
 
 
-@torch_op("aten::permute")
+@torch_op("aten::permute", trace_only=True)
 def aten_permute(self: TTensor, dims: Sequence[int]) -> TTensor:
     """permute(Tensor(a) self, int[] dims) -> Tensor(a)"""
 
+    if not dims:
```

Comment:
`aten_permute` with no dims or dims=None will always be nearest match in SARIF. If we want a perfect match on this, we will need a separate function like `aten_permute_non_dim` to match the no dims situation. But I am not blocking this.

Reply:
dims won't be `None`, it may be empty.